### PR TITLE
Refactor gRPC secured connection to OpenSSL and stub generation formatting fixes

### PR DESCRIPTION
**Pull request by Yasara123**

### CODE REVIEW

File: tests/ballerina-test-integration/src/test/resources/grpc/grpc-secured-unary-service.bal [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+endpoint grpc:Service ep {
+    host:"localhost",
+    port:8085,
+    ssl:{
+            keyStoreFile:"${ballerina.home}/bre/security/ballerinaKeystore.p12",
+            keyStorePassword:"ballerina",
+            certPassword:"ballerina"
+        }
+};
```
Comment (shafreenAnfar) [inline tests/ballerina-test-integration/src/test/resources/grpc/grpc-secured-unary-service.bal]: Can we a have space here :)

File: tests/ballerina-test-integration/src/test/resources/grpc/grpc-secured-unary-service.bal [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+@grpc:serviceConfig {generateClientConnector:false}
+service<grpc:Endpoint> helloWorld bind ep {
+    hello (endpoint client, string name) {
```
Comment (shafreenAnfar) [inline tests/ballerina-test-integration/src/test/resources/grpc/grpc-secured-unary-service.bal]: And also a space about this line..

Comment (Yasara123) [reply tests/ballerina-test-integration/src/test/resources/grpc/grpc-secured-unary-service.bal]: Fixed in 7bee16a

File: tests/ballerina-test-integration/src/test/resources/grpc/grpc-secured-unary-service.bal (hunk 1 above)
Comment (Yasara123) [reply tests/ballerina-test-integration/src/test/resources/grpc/grpc-secured-unary-service.bal]: Fixed in 7bee16a

**Pull request merged**